core/flyout_base.js

```diff
@@ -536,6 +536,22 @@ Blockly.Flyout.prototype.show = function(xmlList) {
         } else {
           gaps.push(default_gap);
         }
+      } else if ((tagName == 'BUTTON') &&
+        (xml.getAttribute('type') === 'status')) {
+        // Scratch extensions can display a status button
+        var extensionId = xml.getAttribute('extensionId');
+        if (extensionId) {
+          var callbackKey = 'STATUS_PRESSED_' + extensionId.toUpperCase();
+          xml.setAttribute('callbackKey', callbackKey);
+          var callback = function(id) {
+            Blockly.statusButtonCallback(id);
+          }.bind(this, extensionId);
```

Comment:
Do you need the extra wrapper? Can this line be:

```
var callback = Blockly.statusButtonCallback.bind(this, extensionId);
```

Or does that not work...?

Reply:
👍